pkg/commands/workload_apply_test.go

```diff
@@ -2950,6 +2951,239 @@ Created workload "spring-petclinic"
 To see logs:   "tanzu apps workload tail spring-petclinic"
 To get status: "tanzu apps workload get spring-petclinic"
 
+`,
+		}, {
+			Name: "git source with non-allowed env var",
+			Prepare: func(t *testing.T, ctx context.Context, config *cli.Config, tc *clitesting.CommandTestCase) (context.Context, error) {
+				os.Setenv("TANZU_APPS_LABEL", "")
```

Comment:
update the test to pass some value instead of empty labels. It clearly shows that value from env var was picked up.

Reply:
Updated

Reply:
can you update this test to pass some value for env var